app/config/FrontendAppConfig.scala

```diff
@@ -19,19 +19,26 @@ package config
 import java.util.Base64
 
 import javax.inject.{Inject, Singleton}
+import config.features.Features
 import play.api.{Configuration, Environment}
 import play.api.Mode.Mode
 import uk.gov.hmrc.play.config.ServicesConfig
-import common.ConfigKeys
+import config.{ConfigKeys => Keys}
 import play.api.mvc.Call
 
 trait AppConfig extends ServicesConfig {
   val analyticsToken: String
   val analyticsHost: String
+  val features: Features
+  val finalReturnPeriodKey: String
   val reportAProblemPartialUrl: String
   val reportAProblemNonJSUrl: String
-  val betaFeedbackUrl: String
-  val betaFeedbackUnauthenticatedUrl: String
+  val vatApiBaseUrl: String
```

Comment:
Many not needed keys and vals in this file too, I assume because it was copied and pasted (please try not to do this - make sure you understand each line of code you're adding as well as follow a TDD approach). The only additional app config key we need at this point is for the vat obligations URL